doc/02-installation.md.d/03-Raspbian.md

```diff
@@ -1,3 +0,0 @@
-# Install Icinga 2 on Raspbian
-<!-- {% set raspbian = True %} -->
```

Comment:
There are some `{% if [...] raspbian %}` remaining in `02-installation.md`. With this removed here, there should be no more need for using it as part of the condition.